src/client/dfuse/dfuse_main.c

```diff
@@ -206,6 +206,11 @@ main(int argc, char **argv)
 		}
 	}
 
+	if (dfuse_info->di_caching and !dfuse_info->threaded) {
```

Comment:
04:26:17  src/client/dfuse/dfuse_main.c: In function ‘main’:
04:26:17  src/client/dfuse/dfuse_main.c:209:29: error: expected ‘)’ before ‘and’
04:26:17    if (dfuse_info->di_caching and !dfuse_info->threaded) {
04:26:17                               ^
04:26:17  scons: *** [build/src/client/dfuse/dfuse_main.o] Error 1
04:26:17  scons: building terminated because of errors.
04:26:17  + rc=2
04:26:17  + '[' 2 '!=' 0 ']'
04:26:17  + echo 'scons failed: 2.'
04:26:17  scons failed: 2.
04:26:17  + set +x
04:26:17  If the error is not in the output above, it might be in the config.log: https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-1777/2/artifact/config.log-centos7-gcc

Reply:
Yes, I'd spotted this locally but haven't pushed a new version yet.  I origionally just pushed what I had before I was out on Friday.